index.bs

```diff
@@ -863,7 +938,7 @@ See [[!HIGHRES-TIME]] for more details.
 <h3 id="error-types">Use Error or DOMException for errors</h3>
 
 Errors in web APIs should be represented as
-<a>ECMAScript error objects</a> (perhaps via the WebIDL {{Error}} type)
+<a>ECMAScript error objects</a> (e.g., [=ECMAScript error objects|Error=])
```

Comment:
Do you mean something like: 

```suggestion
<a>ECMAScript error objects</a> (e.g., {{TypeError}})
```

The aliasing of Error seems strange?

Reply:
Mmm. It moved out of webidl; I've fixed it up and dropped the alias.